lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -4838,62 +4856,68 @@ class JVPEmitter final
   }
 
   /// Handle `tuple_extract` instruction.
-  ///   Original: y = tuple_element_addr x, <n>
-  ///   Tangent: tan[y] = tuple_element_addr tan[x], <n>
-  CLONE_AND_EMIT_TANGENT(TupleElementAddr, teai) {
+  ///   Original: y = tuple_extract x, <n>
+  ///    Tangent: tan[y] = tuple_extract tan[x], <n'>
+  ///                                           ^~~~
```

Comment:
Align ^ with <

Reply:
Done in https://github.com/apple/swift/pull/27269/commits/c0b13d25b1b300e147ab12610b4a6d358d658ee1.